### PR TITLE
Fixing a typo in an incorrect merge conflict resolution

### PR DESCRIPTION
**Pull request by giovannipizzi**
This occurred while fixing some merge conflicts in one of the merges of #69

### CODE REVIEW

File: optimade.md [hunk 1]
```diff
@@ -1322,7 +1322,7 @@ This section defines standard entry types and their properties.
 ### <a name="h.6.2.1">6.2.1. elements</a>
 
 * **Description**: Names of elements found in the structure. 
-* **Requirements/Conventions**: String of chemical symbols of elements as strings as a multi-valued property.
+* **Requirements/Conventions**: Chemical symbols of elements as strings as a multi-valued property.
```
Comment (CasperWA) [inline optimade.md]: Maybe "List of chemical (...)"? Or array?

Comment (giovannipizzi) [reply optimade.md]: Asking for @rartino opinion here - that is his initial text. I also agree with @CasperWA suggestion, but I think this type of sentence is present also elsewhere (there is also a section about multi-valued properties).

**Comment by rartino** [comment]
This is completely re-edited in PR #100. So please accept and merge that one instead of this one. (I'll add a fixes statement of this PR in the description of that PR.)

**Comment by giovannipizzi** [comment]
Superseded by #100 
(I don't think "fixes" works also for PRs, so I close this manually)

**Pull request closed without merging**